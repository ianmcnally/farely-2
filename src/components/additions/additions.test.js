import React from 'react'
import Additions from './additions'
import renderShallow from 'render-shallow'

describe('with required props', () => {
  let component

  beforeAll(() => {
    component = renderShallow(<Additions />).output
  })

  it('renders the component', () => {
    expect(component).toMatchSnapshot()
  })

})

describe('with props.fares', () => {
  const props = {
    fares: [
      {cost: 1.50, rides: 100 },
      {cost: 2.50, rides: 200 }
    ]
  }
  let component

  beforeAll(() => {
    component = renderShallow(<Additions {...props} />).output
  })

  it('renders the fares', () => {
    expect(component).toMatchSnapshot()
  })

})
