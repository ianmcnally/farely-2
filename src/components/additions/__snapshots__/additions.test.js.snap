exports[`with props.fares renders the fares 1`] = `
<ul
  className="list pl0 measure center mt2 mb0">
  <li
    className="lh-copy pv2 ba bl-0 bt-0 br-0 b--dotted b--black-30">
    Add 
    1.50
     for 
    100
     rides
  </li>
  <li
    className="lh-copy pv2 ba bl-0 bt-0 br-0 b--dotted b--black-30 bb-0">
    Add 
    2.50
     for 
    200
     rides
  </li>
</ul>
`;

exports[`with required props renders the component 1`] = `null`;
